packages/mobx-state-tree/src/core/type/type.ts

```diff
@@ -109,10 +129,11 @@ export abstract class ComplexType<C, S, T> implements IType<C, S, T> {
     }
 
     @action
-    create(snapshot: C = this.getDefaultSnapshot(), environment?: any) {
+    create = ((snapshot: C = this.getDefaultSnapshot(), environment?: any) => {
```

Comment:
is there a way to cast a class method to any? I had to convert it to a class property just for this reason

Reply:
solved with ts3 in the pull request about ts3